{
  "uid" : "358bc027e4a0465e",
  "name" : "test_login_1_failed_by_fail_datas",
  "fullName" : "TestCases.test_login.TestLogin#test_login_1_failed_by_fail_datas",
  "historyId" : "1089353f49f9a778d150c7365897a4ef",
  "time" : {
    "start" : 1562162980947,
    "stop" : 1562162980947,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "_pytest.fixtures.FixtureLookupError: ('data', <FixtureRequest for <Function test_login_1_failed_by_fail_datas>>)",
  "statusTrace" : "file F:\\MyProject\\web_ketangpai\\TestCases\\test_login.py, line 42\n       @data(*ld.fail_datas)\n       def test_login_1_failed_by_fail_datas(self, data):\nE       fixture 'data' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, login_web, metadata, monkeypatch, open_url, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, refresh_page, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nF:\\MyProject\\web_ketangpai\\TestCases\\test_login.py:42",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "TestCases"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "DESKTOP-F99FSHQ"
  }, {
    "name" : "thread",
    "value" : "8984-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCases.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "358bc027e4a0465e.json",
  "parameterValues" : [ ]
}